README.md

```diff
@@ -1 +1,53 @@
 # VGG Web
+
+![VGG hero image](https://raw.githubusercontent.com/verygoodgraphics/vgg_web/resources/assets/imgs/vgg_web_container.jpg)
+
+A JavaScript/TypeScript/React/Vue/Svelte and Web Assembly ([WASM](https://developer.mozilla.org/en-US/docs/WebAssembly)) runtime library for [VGG](https://vgg.cool).
+
+This library allows full control over Daruma files with a high-level API for hooking up many simple interactions, as well as a low-level API that allows you to drive your own render loop to create multiple artboards all in one canvas.
+
+## Table of contents
+
+- :star: [VGG Overview](#vgg-overview)
+- 🚀 [Getting Started](#getting-started)
+- :mag: [Supported Browsers](#supported-browsers)
+- :books: [Examples](#examples)
+- :question: [Issues](#issues)
+
+## VGG Overview
+
+[VGG](https://vgg.cool) is the next-gen vector graphics and its toolkit for cross-platform UI development which is compatible with any framework and any platform..
```

Comment:
Change to `VeryGoodGraphics (VGG) is ...`, otherwise it's not clear what VGG is short for. And no hyperlink is needed here because the links are provided below.